app/controllers/application_controller.rb

```diff
@@ -26,9 +27,12 @@ def configure_permitted_parameters
       devise_parameter_sanitizer.for(:account_update) { |u| u.permit(:username, :email, :password, :password_confirmation, :current_password) }
     end
 
+    # refactoring: partips werden nicht in jedem Controller gebraucht
     def partips
+      # inhaltlich: Paginierung/Sortierung, wenn sehr viele partips vorhanden sind?
```

Comment:
Paginating is definitely something that is easily possible and there are quite some gems out there, but an event with so many participations that it requires paginated results is so unlikely, that we don't want to bloat the app before that actually happens.